Backend/lib/yudhishthira/src/Lib/Yudhishthira/Types.hs

```diff
@@ -445,7 +467,15 @@ data RunKaalChakraJobResForUser = RunKaalChakraJobResForUser
   deriving (Show, Read, Generic, ToJSON, FromJSON, ToSchema)
 
 newtype TagNameValue = TagNameValue {getTagNameValue :: Text} -- tagName#tagValue format (only tagName is mandatory)
-  deriving newtype (Show, Read, Eq, ToJSON, FromJSON, ToSchema)
+  deriving newtype (Show, Read, ToJSON, FromJSON, ToSchema)
+
+instance Eq TagNameValue where
```

Comment:
it appears that sometimes we have empty tag values, so added instance to handle it properly `tagName == tagName#`